tsconfig.json

```diff
@@ -3,12 +3,13 @@
     "noEmit": true,
     "lib": ["esnext", "dom"],
     "esModuleInterop": true,
-    "jsx": "preserve",
+    "jsx": "react",
     "strict": true,
     "noImplicitAny": false
   },
   "include": [
     "packages/orbit-components/**/*",
-    "packages/orbit-design-tokens/**/*"
+    "packages/orbit-design-tokens/**/*",
+    "docs/**/*"
```

Comment:
maybe we should create completely different `tsconfig.json` for that workspace? inside docs, like `docs/tsconfig.json`, in order to not change our common and we can make it more strict with `noImpicitAny: true` and other rules 🤔

Reply:
What kind of changes do you think we might need? And do you think we should create separate configurations for other workspaces as well? If we happen to need different `compilerOptions` for `docs` later, e.g. Preact instead of React, then we'll need to split it and deal with any potential complexity that comes with that, but until then I see no need, unless you disagree.

Reply:
yes, i meant that probably we will use different compiler options, since it's new workspace, we can make it more stricter etc, not like our main `tsconfig.json` which we use only for type checking of ts declarations. It's just suggestion

Reply:
Ok, if it turns out we need it, I'll create one.